tests/model/test_time_to_triage.py

```diff
@@ -0,0 +1,96 @@
+import copy
+import datetime
+
+from treeherder.perf.models import PerformanceAlertSummary
+
+
+class TestTimeToClass:
+    def test_alert_summary_created_monday(self, test_perf_alert_summary):
+        test_perf_alert_summary.created = datetime.datetime.fromisoformat('2022-05-30')
+        test_perf_alert_summary.triage_due = None
+
+        assert not test_perf_alert_summary.triage_due
```

Comment:
Lines 9 and 10 will not apply to `test_perf_alert_summary ` until you are calling the save method.